src/mscorlib/shared/System/Collections/Generic/ValueListBuilder.cs

```diff
@@ -43,7 +43,7 @@ public void Append(T item)
             _pos = pos + 1;
         }
 
-        public ReadOnlySpan<T> AsReadOnlySpan()
+        public ReadOnlySpan<T> AsSpan()
```

Comment:
Do we want this change?

Reply:
I think so - we are using `AsSpan` everywhere, this one should not be an exception.

Reply:
I agree and it makes sense. I was just wondering if we need to update any calls to this method as well. 

For instance here: https://github.com/dotnet/corefx/blob/9e0151b48f3a45d497e8835423ed38e7661cd056/src/System.Text.RegularExpressions/src/System/Text/RegularExpressions/RegexWriter.cs#L136

Would it be easier to resolve this separately from the mirror?

Reply:
If you are changing the public surface of share file, it has to be done via the mirror. I do not know how else we would do it.

Reply:
I see. Will we then append the change to the calls in the mirror PR on the corefx side (along with the coreclr version update PR)?

Reply:
Right